core/src/main/scala/org/apache/spark/sql/delta/files/TahoeRemoveFileIndex.scala

```diff
@@ -16,15 +16,13 @@
 
 package org.apache.spark.sql.delta.files
 
-import org.apache.spark.sql.delta.{DeltaErrors, DeltaLog, Snapshot}
+
+import org.apache.hadoop.fs.Path
+import org.apache.spark.sql.SparkSession
```

Comment:
org.apache.hadoop and org.apache.spark are separate import groups. please have a blank line between them, with org.apache.hadoop coming first.